### PR TITLE
Use hook for feature flag.

### PR DESCRIPTION
**Pull request by ronplaymeta**
Hi @pablo-mayrgundter 
This PR uses hook for feature flag checking logic, please review and give me feedback.
Thank you.

### CODE REVIEW

File: src/hooks/useExistInFeature.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import {useEffect, useState} from 'react'
+import {useSearchParams} from 'react-router-dom'
+import debug from '../utils/debug'
+
+
+/**
+ * Feature Hook
```
Comment (pablo-mayrgundter) [inline src/hooks/useExistInFeature.js:7]: `This hook checks for the existence of a named feature in the URL SearchParams (via react-router), e.g. `feature=app,placemark`.`

**Review by pablo-mayrgundter — CHANGES_REQUESTED** [review]
Just one comment suggestion.

Also, please add unit tests for both hooks.

File: src/hooks/useExistInFeature.js (hunk 1 above)
Comment (ronplaymeta) [reply src/hooks/useExistInFeature.js]: done

**Comment by pablo-mayrgundter** [comment]
Per convo, feature flag no longer works for me on the demo.

Please add an e2e test for this.. i'm thinking either a screenshot capture/regression test, or mock three.js methods for adding the mesh or smth earlier than that, just to check that when feature=placemark is enabled and a comment loaded, the placemark call is made

**Comment by ronplaymeta** [comment]
we should login to GitHub to test placemark CRUD, so imho, there is no way to test placemark feature.
anyway, placemark not working issue is fixed, please check again.
ptal

**Pull request merged**